pkg/asset/installconfig/alibabacloud/alibabacloud.go

```diff
@@ -126,6 +129,7 @@ func selectResourceGroup(client *Client) (string, error) {
 				Message: "Resource Group",
 				Help:    "The resource group where the cluster will be provisioned.",
 				Options: options,
+				Default: createNew,
```

Comment:
I suggest that the question asking for the resource group should be removed from the survey questions now that the installer supports creating a new resource group. If the user wants to use an existing resource group, the user should specify that directly in the install-config.yaml.